sw/device/freestanding/README.md

```diff
@@ -0,0 +1,9 @@
+# OpenTitan Freestanding C Headers
+
+This subtree defines the eight headers requred for a C freestanding implementation, as specified in S4.6 of the C11 standard.
```

Comment:
Let's remove the word "eight" as there are nine, and there might be more in future.

Reply:
Done.